README.md

```diff
@@ -36,9 +36,9 @@
 
 ### Some of the recipes you'll find in the Ignite Cookbook:
 
-- [Staying With Expo](https://ignitecookbook.com/docs/recipes/StayingWithExpo)
-- [Pristine Expo Project](https://ignitecookbook.com/docs/recipes/PristineExpoProject)
-- [Add Component Tests to Component Generator](https://ignitecookbook.com/docs/recipes/GenerateComponentTests)
-- [Using Screen Readers](https://ignitecookbook.com/docs/recipes/UsingScreenReaders)
-- [CircleCI CD Setup](https://ignitecookbook.com/docs/recipes/CircleCIRNSetup)
+- [Expo Router](https://ignitecookbook.com/docs/recipes/ExpoRouter/)
```

Comment:
I wonder if this is even a good idea to have in the README, it does seem like it'll keep getting out of date, and it's easy to just go look at the site 🤔

Reply:
Yeah, that's fair! The original thought was to have a preview or CTA of sorts at the end of the README but like you say anyone viewing can see the link in the repo. I'm also good with just removing them to reduce maintenance.

Reply:
What if we add some screenshots of recipes? Then they're frozen but you still get the preview benefit

Reply:
I like the idea of screenshots but I don't know that they'll always be visually appealing, especially on different screen sizes. I've been going back and forth on this, but  what do you think about  removing the links to specific recipes and just putting in a general CTA link to the cookbook? Like this?

> Accelerate your development and dive into the Ignite Cookbook for React Native Today!

Reply:
> especially on different screen sizes

I personally wouldn't worry about this on a README.

But simplifying with a general CTA is also good with me!